packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraTestUtils.m

```diff
@@ -21,6 +21,9 @@
 
   return [[FLTCam alloc] initWithCameraName:@"camera"
                            resolutionPreset:@"medium"
+                                        fps:@(15)
+                               videoBitrate:@(200000)
+                               audioBitrate:@(32000)
```

Comment:
And here.

Where are the actual tests of the new native code?

Reply:
set to `nil`

Reply:
Preparing tests...

Reply:
Unit tests added.